openlibrary/templates/login.html

```diff
@@ -68,13 +68,13 @@ <h1>$_("Log In")</h1>
 
         <div class="formElement bottom">
             <br/>
-            <button value="$_('Log In')" type="submit" class="login-submit larger" name="login" title="$_('Log In')">$_('Log In')</button>
+            <button value="$_('Log In')" type="submit" class="login-submit larger" name="login" title="$_('Log In')">$_("Log In")</button>
         </div>
         <div class="formElement">
             <div class="label"></div>
-            <div class="input smaller"><a href="https://archive.org/account/login.forgotpw.php">$_('Forgot your Password?')</a>
+            <div class="input smaller"><a href="https://archive.org/account/login.forgotpw.php">$_("Forgot your Password?")</a>
             <br/><br/>
-            $:_('Not a member of Open Library? <a href="/account/create">Sign up now</a>.')</div>
+            $_("Not a member of Open Library?") <a href="/account/create">$_('Sign up now</a>.')</div>
```

Comment:
This is likely better as one string. We already have translations for it, so should be good to leave as is to safe our translators some work!

```suggestion
            $:_('Not a member of Open Library? <a href="/account/create">Sign up now</a>.')</div>
```